src/components/providers/components/next-auth.provider.tsx

```
@@ -0,0 +1,7 @@
+"use client";
+
+import { SessionProvider} from "next-auth/react"
+
```

ESLint

[error] Missing semicolon.

(semi)